src/studio-home/card-item/index.tsx

```diff
@@ -162,6 +233,7 @@ type Props = BaseProps & (
  */
 const CardItem: React.FC<Props> = ({
```

Comment:
I'm a bit worried that we are putting too much functionality into this `CardItem` class and its related classes.

Reply:
Yes. In https://github.com/openedx/frontend-app-authoring/pull/2567/commits/09ac876f65f30361c3b10dcd079b1b49748befce I refactored the code to delete all `migrated____` from `CardItem`

About the `select____` props, I think it's best to leave it there, as it's easy to reuse on other screens, because "Select a Card" is not a rare feature for `CardItem`